aszczachor/src/components/about/About.js

```diff
@@ -0,0 +1,7 @@
+import React from "react";
```

Comment:
Nie musisz pobierać wszędzie React, wystarczy, że będzie pobrany w komponencie na samej górze drzewa, jeżeli elint/prettier Ci krzyczy, dodałbym plik konfiguracyjny, który to wyklucza.

Reply:
Tak samo w reszcie komponentów